packages/slice-machine/components/Simulator/components/Header/index.tsx

```diff
@@ -33,13 +33,15 @@ type PropTypes = {
   variation: Models.VariationSM;
   isDisplayEditor: boolean;
   toggleIsDisplayEditor: () => void;
+  disabled: boolean;
 };
 
 const Header: React.FunctionComponent<PropTypes> = ({
   slice,
   variation,
   isDisplayEditor,
   toggleIsDisplayEditor,
+  disabled,
```

Comment:
❓ Struggling to see if this is supposed to disable the whole header. If it's only for the variations selector + switch, should we name it something more specific?